tests/test_oc4ids_tranforms.py

```diff
@@ -0,0 +1,1332 @@
+        },
+    ]
+
+    rationales = "<ocds-213czf-1> A project description\n<ocds-213czf-2> Another project description\n"
+
+    output = transforms._run_transforms(copy.deepcopy(releases), "1", transforms=[transforms.description],)
+    assert output["description"] == rationales
+
+
+def test_description_tender():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "tender": {"description": "A project description"},
+        },
+    ]
+
+    output = transforms._run_transforms(
+        copy.deepcopy(releases), "1", transforms=[transforms.description, transforms.description_tender],
+    )
+    assert output["description"] == releases[0]["tender"]["description"]
+
+
+def test_description_not_tender():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "planning": {"project": {"description": "Another project description"}},
+            "tender": {"description": "A project description"},
+        },
+    ]
+
+    output = transforms._run_transforms(
+        copy.deepcopy(releases), "1", transforms=[transforms.description, transforms.description_tender],
+    )
+    assert output["description"] == releases[0]["planning"]["project"]["description"]
+
+
+def test_environmental_impact():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "planning": {
+                "documents": [
+                    {"id": "doc1", "documentType": "environmentalImpact", "title": "A Document"},
+                    {"id": "doc2", "documentType": "budgetApproval", "title": "Another Document"},
+                ]
+            },
+        },
+    ]
+
+    output = transforms._run_transforms(copy.deepcopy(releases), "1", transforms=[transforms.environmental_impact],)
+    assert output["documents"] == [releases[0]["planning"]["documents"][0]]
+
+
+def test_land_and_settlement_impact():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "planning": {
+                "documents": [
+                    {"id": "doc1", "documentType": "environmentalImpact", "title": "A Document"},
+                    {"id": "doc2", "documentType": "landAndSettlementImpact", "title": "Another Document"},
+                ]
+            },
+        },
+    ]
+
+    output = transforms._run_transforms(
+        copy.deepcopy(releases), "1", transforms=[transforms.land_and_settlement_impact],
+    )
+    assert output["documents"] == [releases[0]["planning"]["documents"][1]]
+
+
+def test_funders_budget():
```

Comment:
Should this test also cover an input with multiple contracting processes, each with different `planning/budget/budgetBreakdown/sourceParty`?